src/main/java/com/dev/cinema/service/impl/MovieServiceImpl.java

```diff
@@ -1,17 +1,19 @@
 package com.dev.cinema.service.impl;
 
 import com.dev.cinema.dao.MovieDao;
-import com.dev.cinema.lib.Inject;
-import com.dev.cinema.lib.Service;
 import com.dev.cinema.model.Movie;
 import com.dev.cinema.service.MovieService;
 import java.util.List;
+import org.springframework.stereotype.Service;
 
 @Service
 public class MovieServiceImpl implements MovieService {
-    @Inject
     private MovieDao movieDao;
```

Comment:
put final here as well to keep code consistent and check other classes for this keyword